Remotion/Web/Development.WebTesting/BrowserLog/IgnoreBrowserLogMessageAttribute.cs

```diff
@@ -42,6 +43,17 @@ public IgnoreBrowserLogMessageAttribute ([NotNull] [RegexPattern] string pattern
     Pattern = pattern;
   }
 
+  public IgnoreBrowserLogMessageAttribute ([NotNull] [RegexPattern] string pattern, [NotNull] [ItemCanBeNull] object?[] templateArgs)
+  {
+    ArgumentUtility.CheckNotNullOrEmpty(nameof(pattern), pattern);
+    ArgumentUtility.CheckNotNullOrEmpty(nameof(templateArgs), templateArgs);
+
+    if (templateArgs.Any())
+      pattern = string.Format(pattern, templateArgs);
+
```

Comment:
```suggestion
```

I would recommend removing this guard:
1. if it can be optimized, .net will optimize
2. in case that pattern is a placeholder text but no arguments are supplied, string-format will fail, which is correct. with the guard, this check is circumvented.